cloud_sql_proxy/README.md

```diff
@@ -0,0 +1,11 @@
+This version of Cloud SQL Proxy has been deprecated. Please use the [official Cloud SQL Proxy](https://github.com/GoogleCloudPlatform/cloudsql-proxy).
+
+
+## Questions?
```

Comment:
Can we remove `Questions` and `Licensing` sections since they are covered in root readme?

Reply:
For sure, let me do that.